SPECIFICATION.md

```diff
@@ -69,10 +69,16 @@ value_with_double_quotes="A value between double quotation marks."
 
 - If a value is empty, it's interpreted as an empty string ''.
 - If a line lacks `=` character, the value is interpreted as an empty string ''.
+- If a file contains 2 or more equal `<name>`-s, then the last value is used.
```

Comment:
May be it will be useful to specify how multiple files will be processed

Reply:
Yep, sure. But only after we add support for multiple files. I guess the first version of the godenv will lack this feature.